Sources/Internal/CoreStoreFetchRequest+CoreStore.swift

```diff
@@ -36,6 +36,6 @@ internal extension CoreStoreFetchRequest {
     @nonobjc
     internal func dynamicCast<U: NSFetchRequestResult>() -> NSFetchRequest<U> {
         
-        return unsafeBitCast(self, to: NSFetchRequest<U>.self)
+        return unsafeDowncast(self, to: NSFetchRequest<U>.self)
```

Comment:
I think using unsafeDowncast would fail in cases where `U` is a subclass of the original `ResultType` generic type of `NSFetchRequest`.